chanzuckerberg/idseq-web
# Update actionview/actionpack gem

# How to reproduce
- Add `gem "actionview", ">= 5.1.6.2"` to Gemfile to test
- Run `bundle update actionview`. See that Consul has a pinned version conflict.
- Run `bundle update actionview consul` to get these changes to Gemfile.lock
- Rebuild web and test the app
